gateway/crates/federated-server/src/server/access_logs.rs

```diff
@@ -1,11 +1,15 @@
 use std::io::Write;
 
 use gateway_config::{AccessLogsConfig, RotateMode};
-use grafbase_telemetry::span::GRAFBASE_TARGET;
+use grafbase_telemetry::metrics::meter_from_global_provider;
 use runtime_local::hooks::{AccessLogMessage, ChannelLogReceiver};
 use tracing_appender::rolling::{RollingFileAppender, Rotation};
 
 pub(crate) fn start(config: &AccessLogsConfig, access_log_receiver: ChannelLogReceiver) -> crate::Result<()> {
+    let log_queue_length = meter_from_global_provider()
+        .i64_up_down_counter("grafbase.gateway.access_log.pending")
```

Comment:
IMHO, too much magic for my taste, supposing that works, to rely on globals to get the same counter, should be part of the `ChannelLogReceiver`